tests/plugins/issue-pr-link/index.js

```diff
@@ -0,0 +1,373 @@
+                        body: "Fix #1 and fix #2 and fix #3 and fix #4 and fix #5",
+                        html_url: "https://github.com/test/repo-test/pull/789",
+                        user: {
+                            login: "contributor"
+                        }
+                    },
+                    repository: {
+                        name: "repo-test",
+                        owner: {
+                            login: "test"
+                        }
+                    }
+                }
+            });
+
+            // Only first 3 issues should be commented on
+            expect(fetchMock.callHistory.called(`${API_ROOT}/repos/test/repo-test/issues/1/comments`, "POST")).toBeTruthy();
+            expect(fetchMock.callHistory.called(`${API_ROOT}/repos/test/repo-test/issues/2/comments`, "POST")).toBeTruthy();
+            expect(fetchMock.callHistory.called(`${API_ROOT}/repos/test/repo-test/issues/3/comments`, "POST")).toBeTruthy();
+            expect(fetchMock.callHistory.called(`${API_ROOT}/repos/test/repo-test/issues/4/comments`, "POST")).toBe(false);
+            expect(fetchMock.callHistory.called(`${API_ROOT}/repos/test/repo-test/issues/5/comments`, "POST")).toBe(false);
+        });
+    });
+
+    describe("pull request edited", () => {
+        test("comments on issue when PR body is edited to reference an issue", async () => {
+            // Mock the issue exists and is open
+            fetchMock.mockGlobal()
+                .get(`${API_ROOT}/repos/test/repo-test/issues/123`, {
+                    status: 200,
+                    body: { state: "open", number: 123 }
+                })
+                .get(`${API_ROOT}/repos/test/repo-test/issues/123/comments`, {
+                    status: 200,
+                    body: []
+                })
+                .post(`${API_ROOT}/repos/test/repo-test/issues/123/comments`, {
+                    status: 200
+                });
+
+            await bot.receive({
+                name: "pull_request",
+                payload: {
+                    action: "edited",
+                    installation: {
+                        id: 1
+                    },
+                    pull_request: {
+                        number: 456,
+                        title: "Resolve the bug",
+                        body: "Fix #123: resolve the bug",
+                        html_url: "https://github.com/test/repo-test/pull/456",
+                        user: {
+                            login: "contributor"
+                        }
+                    },
+                    repository: {
+                        name: "repo-test",
+                        owner: {
+                            login: "test"
+                        }
+                    }
+                }
+            });
+
+            expect(fetchMock.callHistory.called(`${API_ROOT}/repos/test/repo-test/issues/123/comments`, "POST")).toBeTruthy();
+        });
+    });
+});
```

Comment:
```suggestion
});

```

Same here. I think an EOL is needed here.